src/state/index.ts

```diff
@@ -1,35 +1,40 @@
 import { configureStore } from '@reduxjs/toolkit'
 import { setupListeners } from '@reduxjs/toolkit/query/react'
-import { load, save } from 'redux-localstorage-simple'
-import { isTestEnv } from 'utils/env'
+import { persistStore } from 'redux-persist'
 
 import { updateVersion } from './global/actions'
 import { sentryEnhancer } from './logging'
 import reducer from './reducer'
 import { routingApi } from './routing/slice'
 
-const PERSISTED_KEYS: string[] = ['user', 'transactions', 'signatures', 'lists']
+export function createDefaultStore() {
+  return configureStore({
+    reducer,
+    enhancers: (defaultEnhancers) => defaultEnhancers.concat(sentryEnhancer),
+    middleware: (getDefaultMiddleware) =>
+      getDefaultMiddleware({
+        thunk: true,
+        serializableCheck: {
+          // meta.arg and meta.baseQueryMeta are defaults. payload.trade is a nonserializable return value, but that's ok
+          // because we are not adding it into any persisted store that requires serialization (e.g. localStorage)
+          ignoredActionPaths: ['meta.arg', 'meta.baseQueryMeta', 'payload.trade'],
+          ignoredPaths: [routingApi.reducerPath],
+          ignoredActions: [
+            // ignore the redux-persist actions
```

Comment:
Why are we ignoring these actions?

Reply:
https://redux-toolkit.js.org/usage/usage-guide#use-with-redux-persist

this is the instructions for how to integrate redux-persist with redux toolkit